### PR TITLE
fix: start listing parcels near the user

### PR DESCRIPTION
**Pull request by eordano**
This is not a huge improvement -- but it's a starter on prioritizing the download of content that lives closer to the user.

### CODE REVIEW

**Review by menduz — COMMENTED** [review]
Please add a quick comment to the code explaining what those 3 nasty for loops do, ca love letter to our future selves

File: packages/decentraland-loader/lifecycle/lib/scope.ts [hunk 1]
```diff
@@ -11,10 +11,17 @@ export function squareDiff(a: number, b: number) {
 export function parcelsInScope(config: ParcelConfigurationOptions, position: Vector2Component): string[] {
   const result: string[] = []
   const squareRadius = config.lineOfSightRadius * config.lineOfSightRadius
-  for (let x = position.x - config.lineOfSightRadius; x <= position.x + config.lineOfSightRadius; x++) {
-    for (let y = position.y - config.lineOfSightRadius; y <= position.y + config.lineOfSightRadius; y++) {
-      if (squareDiff(x - position.x, y - position.y) <= squareRadius) {
-        result.push(`${x},${y}`)
+  result.push(`${position.x},${position.y}`)
+  for (let dx = 1; dx <= config.lineOfSightRadius; dx++) {
```
Comment (BrianAmadori) [inline packages/decentraland-loader/lifecycle/lib/scope.ts]: We should sort `Vector2` values by squared distance to `position` here and put the results in a temporary list. This code is too illegible.

**Review by BrianAmadori — CHANGES_REQUESTED** [review]

**Comment by eordano** [comment]
Thanks for the input! please review this new version.

>

**Review by menduz — APPROVED** [review]
Sabe

**Review by fmiras — APPROVED** [review]

**Pull request merged**